python/cudf_polars/pyproject.toml

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools",
+    "wheel",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+
+[project]
+name = "cudf-polars"
+dynamic = ["version"]
+description = "Executor for polars using cudf"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license = { text = "Apache 2.0" }
+requires-python = ">=3.9"
+dependencies = [
+    "cudf==24.6.*",
+    "fsspec>=0.6.0",
+    "numpy>=1.23,<2.0a0",
+    "pandas>=2.0,<2.2.3dev0",
```

Comment:
I think we can inherit these from cudf, right? We won't be using any of them directly in cudf.polars AFAIK.

Reply:
Dropped for now.